src/utils/initialization-helper.ts

```diff
@@ -0,0 +1,57 @@
+/*!
+ * Copyright (c) 2019 SAP SE or an SAP affiliate company. All rights reserved.
+ */
+
+export enum InitializationType {
+  freshExpress,
+  existingProject
+}
+
+interface BackendTestScripts {
+  'ci-backend-unit-test': string;
+  'ci-integration-test': string;
+  test?: string;
+}
+
+interface DevDependencies {
+  [name: string]: string;
+}
+
+const expressPackageJsonParts: PackageJsonParts = {
+  backendTestScripts: {
+    'ci-integration-test': 'jest --ci --config=jest.integration-test.config.js',
+    'ci-backend-unit-test': 'jest --ci --config=jest.unit-test.config.js',
+    'test': 'jest'
+  },
+  devDependencies: {
+    'jest': '^24.9.0',
+    'ts-jest': '^24.0.2',
+    '@types/jest': '^24.0.18',
+    'ts-node': '^8.3.0',
+    'tslint': '^5.18.0',
+    'typescript': '^3.5.3',
+    'jest-junit': '^8.0.0'
```

Comment:
As described above, I would not use typescript here. Only devDependencies I see as MUST HAVE are `jest` and `jest-junit`. Adding `eslint` (instead of `tslint`) might be a NICE TO HAVE.